packages/suite/src/utils/suite/tor.ts

```diff
@@ -0,0 +1,6 @@
+import { TOR_DOMAIN } from '@suite-constants/urls';
+
+export const toTorUrl = (url: string) =>
+    url.replace(/https:\/\/([a-z0-9]+\.)trezor.io(.*)/, `http://$1${TOR_DOMAIN}$2`);
```

Comment:
The following takes care also of `foo.bar.trezor.io` case

```suggestion
    url.replace(/https:\/\/(([a-z0-9]+\.)*)trezor.io(.*)/, `http://$1${TOR_DOMAIN}$3`);
```

Reply:
What do you think about this comment, @keraf ?

Reply:
Yep, agreed. I'll update it to your suggestion.